src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
 
 /**
  * @module TextInput
```

Comment:
@unjust See `src/forms/Button.jsx`. Your class will extend `React.PureComponent` instead of `Component`.